### ISSUE TITLE
Improvement: Capitalize should not throw on empty/null strings

### ISSUE THREAD

**Issue by antony** [angus-c/just#169]
Hi - I'm the original author of `just-capitalize`. I make quite a lot of use of it, and I've realised that it might (depending on your thoughts) have an issue with null strings.

If you try `capitalize(someNullString)` it actually throws an error about `cannot call .charAt()` of null.

If you think this is something the library should be handling in a more glamorous way - i.e. simply ignoring the string, I'm happy to open a PR for that.

Right now I do `capitalize(stringName || '')` which isn't ideal.

**Comment by angus-c** [comment]
Hi @antony, I think capitalize of `''` should probably just return `''`

For other types, what I generally do is check the type of the argument and throw with an explanation if it's the wrong type, e.g.

```
function capitalize(str) {
  if (typeof str != 'string') {
    throw new Error('capitalize expects a string as its argument')
  }
```

**Comment by angus-c** [comment]
Assigning to myself to do the above

**Comment by angus-c** [comment]
Fixed in just-capitalize@1.0.1 (#196 )
